plugins/readtimer.koplugin/main.lua

```diff
@@ -19,9 +31,33 @@ function ReadTimer:init()
             text = T(_("Read timer alarm\nTime's up. It's %1 now."), os.date("%c")),
         })
     end
+    self:initEyeSaver()
     self.ui.menu:registerToMainMenu(self)
 end
 
+function ReadTimer:initEyeSaver()
+    self.eyesaver.display_time_clock = self.eyesaver.settings:readSetting("eyesaver_display_time_clock") or ""
+    self.eyesaver.display_timestamp = self.eyesaver.settings:readSetting("eyesaver_timestamp") or 0
+    self.eyesaver.enabled = self.eyesaver.settings:readSetting("eyesaver_enabled") or false
+
+    self.eyesaver.timed_display_callback = function()
```

Comment:
In other places, we call that kind of schedule functions: `something_action` (`_action` helping to distinguish from UI `_callback`).